api/src/main/java/org/pmiops/workbench/actionaudit/targetproperties/TargetPropertyExtractor.kt

```diff
@@ -0,0 +1,48 @@
+package org.pmiops.workbench.actionaudit.targetproperties
+
+import org.pmiops.workbench.model.Profile
+import org.pmiops.workbench.model.Workspace
+import kotlin.reflect.KClass
+
+class TargetPropertyExtractor {
+    companion object {
+
+        @JvmStatic
+        fun <T : Any, E : TargetProperty<T>> getPropertyValuesByName(
```

Comment:
So, does this return a sub object with just the values specified in the array? If so:
Nit: I'd call it: `getPropertyValuesByNames` and then name the variable `names` or maybe `propertyNames`

Although it looks like its maybe pulling a specific Property out of a store somewhere? If that's the case, I would probably try to make that a little more clear here.